lib/pronto/eslint.rb

```diff
@@ -37,8 +37,10 @@ def options
     def new_message(offence, line)
       path = line ? line.patch.delta.new_file[:path] : '.eslintrc'
       level = line ? :warning : :fatal
+      message = offence['message']
+      message.insert(0, "#{offence['ruleId']}: ") if offence['ruleId']
```

Comment:
```suggestion
      message = "#{offence['ruleId']}: #{message}" if offence['ruleId']
```
Reads much simpler to me.